### DISCUSSION TITLE
BLAS/LAPACK Survey

### DISCUSSION THREAD

**Discussion by sebrowne** [trilinos/Trilinos#13603] [Polls]
We are exploring changing the code that finds/registers the linear algebra libraries (BLAS/LAPACK) in Trilinos' build system to deliver a better user experience.  Please let us know which of the following linear algebra libraries you use when building Trilinos so that we can identify cases that we should test (we can't test everything, but ideally we would like to test the most common user workflows).

**Comment by ndellingwood** [comment]
We use OpenBlas, as well as Netlib, Intel MKL, and Arm Performance Libraries

**Comment by jhux2** [comment]
@sebrowne Can't select more than one: I use OpenBLAS, MKL, Cray LibSci, sometimes Netlib.

**Comment by lucbv** [comment]
OpenBLAS, MKL, ArmPL, libSci and also BLIS which is not on the list yet great because it does not require a license (MKL and libSci aren't free) also it does not require a FORTRAN compiler so it's pretty much the only option on stock OSX.
